Src/FluentAssertions/Equivalency/IValidateChildNodeEquivalency.cs

```diff
@@ -1,9 +1,9 @@
 namespace FluentAssertions.Equivalency;
 
-public interface IEquivalencyValidator
+public interface IValidateChildNodeEquivalency
 {
     /// <summary>
     /// Runs a deep recursive equivalency assertion on the provided <paramref name="comparands"/>.
     /// </summary>
-    void RecursivelyAssertEquality(Comparands comparands, IEquivalencyValidationContext context);
+    void AssertEquivalencyOf(Comparands comparands, IEquivalencyValidationContext context);
```

Comment:
I do like this renaming.

Reply:
Consider let out the re-renaming of `...Equality` to `...Equivalency`, because this more correct, I think...